src/av2/structures/bev_params.py

```diff
@@ -0,0 +1,57 @@
+# <Copyright 2022, Argo AI, LLC. Released under the MIT license.>
+
+"""Defines a helper class for generating bird's eye view (BEV) images."""
+
+import math
+from typing import Tuple
+
+import numpy as np
+
+from av2.geometry.sim2 import Sim2
+from av2.utils.typing import NDArrayFloat
+
+
+class BEVParams:
```

Comment:
Do you think we could merge functionality with this? https://github.com/argoai/av2-api/blob/082b9b9bc59f29b13d44da6800fd3063adf59f37/src/av2/structures/ndgrid.py#L114